### PR TITLE
[Merged by Bors] - fix: display fetch status

### LINKED ISSUE #2862: [Bug]: smdk build can appear to hang if cargo is updating index
**What happened**
smdk build does not relay cargo index fetch

**Expected behavior**
show cargo activity so user has transparency about activity

**Describe the setup**
performing smdk build

**How to reproduce it (as minimally and precisely as possible)**
smdk generate <project>
cd project
smdk build. # but this also requires a large cargo fetch update to occur

**Log output**
n/a

**Environment (please complete the following information):**
Linux or Max

**Additional context**
diagnosis w/ cargo build showed long Fetching... update

### PR DESCRIPTION
**Pull request by davidbeesley**
fixes  #2862

### CODE REVIEW

File: crates/cargo-builder/src/package.rs [hunk 1]
```diff
@@ -33,6 +33,7 @@ impl PackageInfo {
         let metadata = MetadataCommand::new()
             .manifest_path(&current_project)
             .features(CargoOpt::AllFeatures)
+            .verbose(true)
```
Comment (galibey) [inline crates/cargo-builder/src/package.rs:36]: Is it enough? It affects only `cargo metadata` command, not `cargo build`

Comment (davidbeesley) [reply crates/cargo-builder/src/package.rs:36]: `cargo build` was already giving output, it was just `metadata` command that was not.

**Review by galibey — APPROVED** [review]

**Pull request closed without merging**